pythonFlu_INSTALLNOTES

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
We should convert these instructions into a `.travis.yml` files so that we can run the travis-ci continuous testing on the repository. Please let me know if you need help on this.